services/job_chat/job_chat.py

```diff
@@ -20,6 +20,21 @@
 
 logger = create_logger("job_chat")
 
+def send_status(message: str):
```

Comment:
This stuff will need to be shared by multiple services - so we need to abstract this out into a higher level util function. We might need a helper which accepts the logger (or we just pass it in every time)